test/unix/test_lwt_bytes.ml

```diff
@@ -93,7 +100,7 @@ let test_wait_mincore buff_len offset =
   >>= fun () ->
```

Comment:
These next four lines should be redundant now.

Reply:
yes